v3/generator/templates/resource.tmpl

```diff
@@ -0,0 +1,35 @@
+package {{ .Package }}
+
```

Comment:
We could add a note like THIS CODE IS GENERATED or similar.

Reply:
That's a good point and good practice. I'll add a note.

Reply:
```
// Code generated by v3/generator; DO NOT EDIT.
```

Reply:
Done